x-pack/qa/sql/security/build.gradle

```diff
@@ -23,14 +23,30 @@ subprojects {
     testCompile "org.elasticsearch.plugin:x-pack-core:${version}"
   }
 
+  task changeAuditLoggingRollingPolicy {
+    def toConcatenate = files("${project(':distribution').projectDir}/src/config/log4j2.properties", "${project(':x-pack').projectDir}/plugin/core/src/main/config/log4j2.properties")
+    def outputFileName = "${buildDir}/log4j2.properties"
+    def output = new File(outputFileName)
```

Comment:
It's better to use `file()` or `project.file()` instead of `new File` in Gradle as the former always resolves relative paths to project dir. 
It won't make a difference here, but a best practice that we would like to follow.